<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_NoteFinding</class>
 <widget class="QWidget" name="Game_NoteFinding">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game_NoteFinding</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.466667, y2:1, stop:0 rgba(72, 158, 191, 255), stop:0.466667 rgba(180, 214, 227, 255), stop:1 rgba(72, 157, 190, 255));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="game">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGraphicsView" name="graphicsView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>2000</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QProgressBar" name="score">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {
     border: 2px solid grey;
	  background-color: rgba(180, 214, 227, 255);
   
 }

QProgressBar::chunk {
     background-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.442857, y2:1, stop:0 rgba(191, 72, 72, 255), stop:0.995238 rgba(227, 108, 108, 255));
 }
</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="guiChallenge">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(180, 214, 227, 255);
border: 10px solid;
border-top-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.442857, y2:1, stop:0 rgba(72, 158, 191, 255), stop:0.995238 rgba(180, 214, 227, 255));
border-right-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.442857, y2:1, stop:0 rgba(180, 214, 227, 255), stop:0.995238 rgba(72, 158, 191, 255));
border-left-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.442857, y2:1, stop:0 rgba(180, 214, 227, 255), stop:0.995238  rgba(72, 158, 191, 255));
border-bottom-color: qlineargradient(spread:pad, x1:0.438, y1:0, x2:0.442857, y2:1, stop:0 rgba(72, 158, 191, 255), stop:0.995238 rgba(180, 214, 227, 255));</string>
           </property>
           <property name="text">
            <string>Note: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="won">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>110</y>
           <width>201</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>YOU WON!</string>
         </property>
        </widget>
        <widget class="QPushButton" name="playAgain">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>210</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0.097, x2:1, y2:0, stop:0 rgba(175, 175, 175, 255), stop:1 rgba(255, 255, 255, 255))</string>
         </property>
         <property name="text">
          <string>Play again</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
